### PR TITLE
RHIDP-1764: Document minimum resource requirements for Operator

### PR DESCRIPTION
**Pull request by hmanwani-rh**
**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):**
1.3

**Issue:**
[RHIDP-1764](https://issues.redhat.com/browse/RHIDP-1764)

**Link to docs preview:**
Find the link within this PR

**Reviews:**
- [x] SME: @rm3l 
- [x] QE: @zdrapela 
- [x] Docs review: @themr0c 

**Additional information:**

### CODE REVIEW

**Comment by hmanwani-rh** [comment]
@rm3l, @themr0c Please review this PR and share your approval

File: modules/getting-started/ref-rhdh-sizing.adoc [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
 
 |Central Processing Unit (CPU)
 |4 vCPU
 |2 vCPU
+|1 vCPU
 
 |Memory
 |16 GB
 |8 GB
+|256 Mi
```
Comment (rm3l) [inline modules/getting-started/ref-rhdh-sizing.adoc:22]:
```suggestion
|1500 Mi
```

Based on what we have in https://github.com/redhat-developer/rhdh-operator/blob/main/.rhdh/bundle/manifests/rhdh-operator.clusterserviceversion.yaml#L283, the memory limit of the controller container has been increased to 1 Gi. Taking into account the kube-rbac-proxy container limit set to [128 Mi](https://github.com/redhat-developer/rhdh-operator/blob/main/.rhdh/bundle/manifests/rhdh-operator.clusterserviceversion.yaml#L239), I think we should update this.

File: modules/getting-started/ref-rhdh-sizing.adoc [hunk 2]
```diff
@@ -1,28 +1,33 @@
...
 
 |Storage size
 |2 GB
 |20 GB
+|NA
 
 |Replicas
 |2 or more
 |3 or more
+|NA
```
Comment (rm3l) [inline modules/getting-started/ref-rhdh-sizing.adoc:32]:
```suggestion
|1
```

I would say exactly 1 replica, as the operator is cluster-wide atm, and we want to have a single instance of it running.

File: modules/getting-started/ref-rhdh-sizing.adoc [hunk 3]
```diff
@@ -1,28 +1,33 @@
...
 
 |Memory
 |16 GB
 |8 GB
+|256 Mi
 
 |Storage size
 |2 GB
 |20 GB
+|NA
```
Comment (rm3l) [inline modules/getting-started/ref-rhdh-sizing.adoc:27]:
```suggestion
|50 Mi
```

The operator needs some ephemeral storage - see https://github.com/redhat-developer/rhdh-operator/blob/main/.rhdh/bundle/manifests/rhdh-operator.clusterserviceversion.yaml#L282

**Review by rm3l — CHANGES_REQUESTED** [review]

**Review by rm3l — APPROVED** [review]

**Review by zdrapela — APPROVED** [review]

File: modules/getting-started/ref-rhdh-sizing.adoc [hunk 4]
```diff
@@ -1,28 +1,33 @@
...
 
 |Storage size
 |2 GB
 |20 GB
+|50 Mi
 
 |Replicas
 |2 or more
 |3 or more
+|1
```
[the remark this answers is not in the document]
Comment (hmanwani-rh) [reply modules/getting-started/ref-rhdh-sizing.adoc]: Done

**Review by themr0c — APPROVED** [review]

**Pull request merged**